meta-iotqa/conf/test/refkit-image-common-extensive.manifest

```diff
@@ -1 +1,3 @@
 # Extensive tests for common profile
+oeqa.runtime.bluetooth.comm_bt_command
```

Comment:
@jairglez actually, adding it in packagegroup-common-test.bb would probably be the best option. the other alternative is to do it like @ipuustin mentioned but packagegroup-common-test.bb makes toggling all test content on/off easier.